classes/cluster/mk20_stacklight_advanced/openstack/compute.yml

```diff
@@ -3,6 +3,7 @@ classes:
 - system.linux.system.repo.mos8
 - system.nova.compute.cluster
 - system.opencontrail.compute.cluster
+- service.opencontrail.compute.cluster
```

Comment:
yes, this is tricky, it can stay here, until stacklight is merged and then we will clean it on cluster/system level